account_invoice_kanban/models/account_invoice.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class AccountInvoice(models.Model):
+    _name = 'account.invoice'
+    _inherit = ['account.invoice', 'base.kanban.abstract']
+    _order = 'date_due asc'
+
+    @api.multi
+    def _read_group_stage_ids(self, stages, domain, order):
```

Comment:
Oops I added this to a collapsed thread:

This can be removed now due to OCA/server-tools#989